### PR TITLE
Feature/mui datatables

### PR DESCRIPTION
**Pull request by azeng4499**
- changed farm values table to material-table
- removed mui-datatable version
- made farm values filter mutli select

### CODE REVIEW

**Comment by mikahpinegar** [comment]
i'm unable to filter on affiliations. the little modal type thing with checkboxes opens but they're unclickable. i really love the styling though this is an awesome change i think

File: src/Biomass/FarmValuesTable.js [hunk 1]
```diff
@@ -132,174 +69,342 @@ const FarmValuesTable = ({
...
+    },
+    {
+      field: 'uncorrected_cc_dry_biomass_kg_ha',
+      title: 'Uncorrected Dry Weight',
+      type: 'numeric',
+      render: (rowData) => {
+        return rowData.uncorrected_cc_dry_biomass_kg_ha
+          ? units === 'kg/ha'
+            ? Math.round(rowData.uncorrected_cc_dry_biomass_kg_ha)
+            : Math.round(rowData.uncorrected_cc_dry_biomass_kg_ha * 0.892179)
```
Comment (mikahpinegar) [inline src/Biomass/FarmValuesTable.js:114]: do we want these coefficients extended @brianwdavis?

Comment (brianwdavis) [reply src/Biomass/FarmValuesTable.js:114]: Extended to what? I think rounding to the nearest integer on something in the 100-10,000 order is good.

**Comment by azeng4499** [comment]
> i'm unable to filter on affiliations. the little modal type thing with checkboxes opens but they're unclickable. i really love the styling though this is an awesome change i think

Ah, it might be because I disabled all the other checkboxes if All is selected (All is selected by default). On second thought, this might be confusing to users?

File: src/Biomass/FarmValuesTable.js (hunk 1 above)
Comment (brianwdavis) [reply src/Biomass/FarmValuesTable.js:114]: 4 digits is plenty for this purpose

**Comment by mikahpinegar** [comment]
> > i'm unable to filter on affiliations. the little modal type thing with checkboxes opens but they're unclickable. i really love the styling though this is an awesome change i think
> 
> Ah, it might be because I disabled all the other checkboxes if All is selected (All is selected by default). On second thought, this might be confusing to users?

it seems like you want to unselect all the other checkboxes, not disable them

File: src/Biomass/FarmValuesTable.js (hunk 1 above)
Comment (azeng4499) [reply src/Biomass/FarmValuesTable.js:114]: yes i can

**Comment by azeng4499** [comment]
> > > i'm unable to filter on affiliations. the little modal type thing with checkboxes opens but they're unclickable. i really love the styling though this is an awesome change i think
> > 
> > 
> > Ah, it might be because I disabled all the other checkboxes if All is selected (All is selected by default). On second thought, this might be confusing to users?
> 
> it seems like you want to unselect all the other checkboxes, not disable them

gotcha, getting on it

**Comment by mikahpinegar** [comment]
> > > > i'm unable to filter on affiliations. the little modal type thing with checkboxes opens but they're unclickable. i really love the styling though this is an awesome change i think
> > > 
> > > 
> > > Ah, it might be because I disabled all the other checkboxes if All is selected (All is selected by default). On second thought, this might be confusing to users?
> > 
> > 
> > it seems like you want to unselect all the other checkboxes, not disable them
> 
> gotcha, getting on it

and also unselect all if any other checkbox is selected as well. so either it's all and no other checkbox or any other checkbox and not all

**Comment by mikahpinegar** [comment]
also there's a merge conflict, can you merge dev into your branch and resolve it @azeng4499?

**Comment by azeng4499** [comment]
> also there's a merge conflict, can you merge dev into your branch and resolve it @azeng4499?

it was a bunch of stuff regarding the package.json. I just kept everything on dev to be safe

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**